modules/AM_ProjectTemplates/language/en_us.lang.php

```diff
@@ -142,5 +142,8 @@
 	'LBL_SCHEDULING_FORM_TITLE' => 'Resource List',
 	'LBL_NONE' => 'None',
 
+	'LBL_AOS_PRODUCTS_AM_PROJECTTEMPLATES_1_FROM_AM_PROJECTTEMPLATES_TITLE' => 'Product: Project Templates from Project Templates Title',
+	'LBL_AM_PROJECTTEMPLATES_PROJECT_1_FROM_PROJECT_TITLE' => 'Project Templates Project from Project Title',
```

Comment:
not a code issue... only a translator issue.
On Crowdin we will only have to translate one sentence as the other get auto-filled (if they are 100% the same)